### PR TITLE
Generate the CAIDA data timestamp based on the current month

### PR DESCRIPTION
**Pull request by hellais**
Adds support for getting always the latest CAIDA AS name data

### CODE REVIEW

File: generator_ripe_caida.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ set -exuo pipefail
 # Use cpu-efficient tools across 2-4 cores
 
 tstamp=$(date +%Y%m%d%H%M%S)
-caida_tstamp=20201001
+caida_tstamp=$(date +%Y%m%01)
```
Comment (bassosimone) [inline generator_ripe_caida.sh:11]: This would break because they don't update monthly. We need a smarter logic here.

**Review by bassosimone — CHANGES_REQUESTED** [review]

File: generator_ripe_caida.sh (hunk 1 above)
Comment (bassosimone) [reply generator_ripe_caida.sh:11]: See http://data.caida.org/datasets/as-organizations/

Comment (hellais) [reply generator_ripe_caida.sh:11]: Good call. I will take another stab at this.

Comment (FedericoCeratto) [reply generator_ripe_caida.sh]: Even if we were to update monthly, we need to manage race conditions around the time of publishing from Caida

Comment (hellais) [reply generator_ripe_caida.sh]: How frequently is the github action run?

Comment (bassosimone) [reply generator_ripe_caida.sh]: Weekly: https://github.com/ooni/asn-db-generator/blob/master/.github/workflows/generate.yml#L5

Comment (bassosimone) [reply generator_ripe_caida.sh]: (After a few weeks of inactivity, GH disables automatic runs)

**Comment by bassosimone** [comment]
@hellais "this should work until 2099" makes me think of:

Spider-Man_2099_Vol_1_39

**Review by bassosimone — APPROVED** [review]
🐳 

This is an improvement, thank you for committing it. I vote for merging and we'll see how frequently we have conflicts. In case of conflicts the build fails, so we'll notice.

**Pull request merged**